asab/library/service.py

```
@@ -382,6 +388,94 @@ async def _list(self, path, providers):
+		try:
+			fav_data = yaml.load(fav_file, Loader=yaml.CSafeLoader)
+		except Exception:
+			L.exception("Failed to parse '/.favorites.yaml'")
+			self.Favorites = {}
+			self.FavoritePaths = []
+			return
+		finally:
+			try:
+				fav_file.close()
+			except Exception:
+				pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)